src/member.rs

```diff
@@ -368,37 +368,37 @@ where
         let su = match uu.check(self.keybox) {
             Ok(su) => su,
             Err(uu) => {
-                debug!(target: "AlephBFT-member", "{:?} Wrong signature received {:?}.", self.index(), &uu);
+                trace!(target: "AlephBFT-member", "{:?} Wrong signature received {:?}.", self.index(), &uu);
```

Comment:
This is somewhat controversial, because many other incorrect-message errors are logged as warn.

Reply:
Good catch, no idea what I've been thinking, fixed.